src/ripple/basics/BasicConfig.h

```diff
@@ -76,6 +76,38 @@ class Section
         return values_;
     }
 
+    /**
+     * Set the legacy value for this section.
+     */ 
+    void
+    legacy (std::string value)
+    {
+        if (lines_.empty ())
+            lines_.emplace_back (std::move (value));
+        else
+        {
+            assert (lines_.size () == 1);
+            lines_[0] = std::move (value);
+        }
+    }
+
+    /**
+     * Get the legacy value for this section.
+     *
+     * @return The retrieved value. A section with an empty legacy value returns
+               an empty string.
+     */
+    std::string
+    legacy () const
+    {
+        if (lines_.empty ())
+            return "";
+        else if (lines_.size () > 1)
```

Comment:
Sorry, a few more more cosmetic details here.

1. Never use an `else` after `return`, `continue` or `break` (no need!)
2. Even though that's one logical line on 106, it breaks across two actual lines, so you should consider using braces?